presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java

```diff
@@ -64,7 +72,7 @@ public class HiveColumnHandle
 
     private static final String UPDATE_ROW_ID_COLUMN_NAME = "$shard_row_id";
 
-    // Ids <= this can be used for distinguishing between different prefilled columns.
+    // Negative IDs < this can be used for distinguishing between different hidden columns.
```

Comment:
I think less than is confusing for negative numbers when used in a non-technical way. it's hard to know if someone means a larger negative number (mathematically less than) or a smaller negative number (lower in magnitude), even though those are completely opposite meanings. Let's be more explicit to remove the ambiguity e.g.
```suggestion
    // Negative IDs lower in magnitude than this can be used for distinguishing between different hidden columns (-1 <= id < this).
```

Reply:
i agree they are well defined, but the comment as written is incorrect.  It should be "negative ids > this can be used for distinguishing hidden columns".  Given that it was incorrect originally, and incorrect after your changes, I expect it would be a common mistake and we should change the wording (doesn't have to be exactly what i suggested, but anything that would let someone reading it quickly to not make the same mistake, e.g. just specifying the range of -1 through -12)